document/Definitions.mdk

```diff
@@ -12,14 +12,17 @@ file
     the activity of programming
   ~ Example: Source code, object code, program configuration data, documentation.
 
-issue
-  ~ condition present in a programming artifact which has the potential to detract from the quality of the program
+result
+  ~ condition present in a programming artifact
+
+problem
+  ~ result which indicates a condition that has the potential to detract from the quality of the program
   ~ Example: A security vulnerability,
     a deviation from conformance to contractual or legal requirements,
     a deviation from conformance to stylistic standards.
 
 (static analysis) tool
-  ~ program that examines programming artifacts in order to detect and report potential issues, without executing the program
+  ~ program that examines programming artifacts in order to report results, without executing the program
```

Comment:
i'd use 'detect problems' here since this is the express purpose of the tool. it can be implied that a tool will emit logging details, report internal errors, etc.